src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl

```diff
@@ -14897,3 +14897,41 @@ voag:QUDT-QUANTITY-KINDS-VocabCatalogEntry_v1.2
   rdfs:label "QUDT Quantity Kinds Vocabulary Catalog Entry v1.2" .
 
 
+# new quantity kinds
+
+quantitykind:spectralIrradiance
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit of Spectral Irradiance";
+  qudt:plainTextDescription "Spectral Irradiance is a radiometry term for the power per unit area of electromagnetic radiation per unit wavelength incident at a surface." ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
+  qudt:hasDimensionVector qkdv:A0E0L-4I0M1H0T-3D0 .
+
+quantitykind:spectralEmittance
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit of Spectral Emittance";
+  qudt:plainTextDescription "Spectral Emittance is a radiometry term for the power per unit area of electromagnetic radiation unit wavelength emerging from a surface." ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
+  qudt:hasDimensionVector qkdv:A0E0L-4I0M1H0T-3D0 .
+  
+quantitykind:spectralRadiance
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit of Spectral Radiance";
+  qudt:plainTextDescription "Spectral Radiance is a radiometry term for the power per unit area of electromagnetic radiation per unit wavelength per unit solid angle emerging from a surface." ;
+  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D-1;
```

Comment:
Invalid dimension vector

Reply:
Fixed in 15d1360fefa7f2bec2ba499ee6f4f217a005a85e assuming the problem was the D-1

Reply:
I'm still seeing 

qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D-1;

Reply:
Hopefully completely fixed in 38682b9